.github/workflows/publish.yml

```diff
@@ -1,23 +1,20 @@
 name: CI
 
-on: [push, pull_request]
+on: [push]
 
 jobs:
   default:
     name: Publish CI tagged release for MDX
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # for lerna
```

Comment:
This should make checkout out much slower. Why is it needed?

Reply:
`lerna` requires commit history

Reply:
https://stackoverflow.com/questions/57597367/how-to-set-up-github-actions-to-publish-a-lerna-monorepo

For `checkout@v1`, it pulls all commits, for `checkout@v2`, it doesn't pull all commits by default.

Reply:
does it require everything all the way back to the start? Could we limit it?

Reply:
I don't know is there any other best practice for it.

Let's keep discussion at https://github.com/mdx-js/mdx/pull/1498#discussion_r601604403